src/Client/OAuth2Client.php

```diff
@@ -136,9 +145,17 @@ public function getOAuth2Provider()
     }
 
     /**
+     * @return bool
+     */
+    public function isStateless()
+    {
+        return $this->isStateless;
+    }
```

Comment:
Same here - this is an unnecessary extension point, until someone gives us a good reason :)